### PR TITLE
bug fixes solver.py

### PR DESCRIPTION
**Pull request by marcrawson**
it's just bug fixes

### CODE REVIEW

File: solver.py [hunk 1]
```diff
@@ -11,11 +13,14 @@ def can_be(self, number):
...
     def __repr__(self):
         return str(self.possible[4])
+# So above you'll find our cell, we can ignore that
+# (just a formality)
     
 #troubleshooting method that displays the board according to the __repr__ method in Cell class
 def display_board():
     for cell in board:
         print(cell)
+# i don't know why anyone would use this function lol
```
Comment (davenfroberg) [inline solver.py:23]: world rudest comment

File: solver.py [hunk 2]
```diff
@@ -50,6 +56,13 @@ def create_board():
         
         sub_board.append(row)
     return sub_board
+# if the last function didn't this one definitely did
+
+def greeting():
+    while(1):
```
Comment (davenfroberg) [inline solver.py:62]: this seems like it might break the program

File: solver.py [hunk 3]
```diff
@@ -76,8 +89,9 @@ def solve(row, column, number):
 #access board by [row][column]
 board = create_board()
 
+greeting()
 create_board()
 
 solve (1,2,4)
 display_board()
-
+# consider yourself taken care of
```
Comment (davenfroberg) [inline solver.py:97]: thx!

**Review by davenfroberg — CHANGES_REQUESTED** [review]

File: solver.py (hunk 1 above)
Comment (marcrawson) [reply solver.py:23]: It's constructive

File: solver.py (hunk 2 above)
Comment (marcrawson) [reply solver.py:62]: It shouldn't be an issue sir.

File: solver.py (hunk 3 above)
Comment (marcrawson) [reply solver.py:97]: It's just an exposé sir.

**Comment by marcrawson** [comment]
I think you made a mistake sir

**Pull request closed without merging**